utils/notification_service.py

```diff
@@ -1043,6 +1043,7 @@ def prepare_reports(title, header, reports, to_truncate=True):
         "PyTorch pipelines": "run_tests_torch_pipeline_gpu",
         "TensorFlow pipelines": "run_tests_tf_pipeline_gpu",
         "Torch CUDA extension tests": "run_tests_torch_cuda_extensions_gpu_test_reports",
+        "Quantization tests": "run_tests_quantization_torch_gpu",
```

Comment:
should be revised if we decide to move quantization tests outside this workflow file.